crates/meilisearch/Cargo.toml

```diff
@@ -57,7 +57,7 @@ meilisearch-types = { path = "../meilisearch-types" }
 mimalloc = { version = "0.1.43", default-features = false }
 mime = "0.3.17"
 num_cpus = "1.16.0"
-obkv = "0.2.2"
+obkv = { git = "https://github.com/kerollmops/obkv", branch = "unsized-kvreader" }
```

Comment:
Should we:
- revert,
- stay on that branch
- make a new release?

Reply:
[In the _During Prerelease_](https://github.com/meilisearch/meilisearch/issues/4985) section: I must release a new (breaking) version of the obkv crate.